### ISSUE TITLE
Open LazyBox From Bytes

### ISSUE THREAD

**Issue by re-bola** [hivedb/hive#177]
Seems like the full implementation is there, it just needs to be passing the bytes open to the base _openBox from _openLazyBox. Any reason why that can't be done? Happy to put out the PR myself if not.

**Comment by leisim** [comment]
The reason I did not implement a `LazyMemoryBox` is that it makes no sense in my option.

Either the box is lazy (e.g. read from disk on demand) or not. A lazy box in memory makes no sense.

**Comment by re-bola** [comment]
Fair enough. I guess what I want is to open a LazyBox from assets/ without having to copy the hive file to disk. Is there a way to do that?

**Comment by leisim** [comment]
Why don't you just use a normal box? You don't get any advantages from using an in-memory lazy box and you have to deal with the async code.

To answer your question: It is not possible currently and I still don't really see the use case.
